src/components/common/ChildCard.js

```diff
@@ -22,7 +22,7 @@ function NewChildCard(props) {
   };
 
   return (
-    <div className="newChildCardContainer">
+    <div classsName="newChildCardContainer">
```

Comment:
🤔 there's an extra 's' in classsName.

Reply:
It just might not be meant to be Dennis!! 😭😂

Reply:
Fixed it~!!!!